### ISSUE TITLE
Add support for interactive registration

### ISSUE THREAD

**Issue by K-Meech** [mobie/mobie-viewer-fiji#411]
It would be great to support different registration methods in MoBIE e.g.
- manual registration
- bigwarp
- elastix

**Comment by constantinpape** [comment]
Potentially related issue: https://github.com/mobie/mobie.github.io/issues/32

**Comment by tischi** [comment]
@NicoKiaru
What's the status of the BigWarp integration with BDV-Playground?
Could we just enable it in MoBIE?
If so, could you point me to the relevant code to do so?

**Comment by NicoKiaru** [comment]
You can look for "Launch BigWarp" in Fiji's search bar.

The command:

https://github.com/bigdataviewer/bigdataviewer-playground/blob/master/src/main/java/sc/fiji/bdvpg/scijava/command/source/BigWarpLauncherCommand.java

The 'action':

https://github.com/bigdataviewer/bigdataviewer-playground/blob/master/src/main/java/sc/fiji/bdvpg/sourceandconverter/register/BigWarpLauncher.java

**Comment by tischi** [comment]
@NicoKiaru Thanks a lot!
This is the output, right?
```
BigWarpLauncher bwl = new BigWarpLauncher(movingSacs, fixedSacs, bigwarpname, converterSetups);
// ...   
// Output
gridsource = bwl.getGridSource();
warpmagnitudesource = bwl.getWarpMagnitudeSource();
warpedsources = bwl.getWarpedSources();
```

Is there some way to only enable affine transform (or only rigid) registration in bigWarp and then get back the affine transform?

**Comment by NicoKiaru** [comment]
> In fact, in my use-case the movingSacs are TransformedSource already and I think it would be OK if their sourceTransform was directly modified by the registration.

Yes then you can probably get the transformed source, get its transform, and build a new Transformed source out of it ?

**Comment by tischi** [comment]
@NicoKiaru 
What is this doing?
```
		// Output bdvh handles -> will be put in the object service
		bdvhq = bigWarpLauncher.getBdvHandleQ();
		bdvhp = bigWarpLauncher.getBdvHandleP();

		bdvDisplayService.pairClosing(bdvhq, bdvhp);
```

**Comment by tischi** [comment]
Wohoooo!

<img width="500" alt="image" src="https://user-images.githubusercontent.com/2157566/150537446-74aaa2b0-8090-4e4d-9efe-0ca578ce06fd.png">

@NicoKiaru @bogovicj Amazing! Took me just ~45 minutes to make BigWarp work in MoBIE 🥳 
@martinschorb @constantinpape

**Comment by tischi** [comment]
@bogovicj what would be cool now would be a way for the user to "Cancel" in a way that MoBIE knows that it should not apply the transformation and revert back to the original `SourceTransform`.
So, I would need something in BigWarp that says "Cancel" and that I could listen to.
Or, I guess we could have an own little window pop up in MoBIE with two buttons: "Apply transform in MoBIE" and "Cancel". What do you think?

**Comment by tischi** [comment]
@NicoKiaru @bogovicj When launching BigWarp the second time, I have more sources (`Bigwarp *` in the source tree, do you know why they are there and how I could avoid this?

<img width="530" alt="image" src="https://user-images.githubusercontent.com/2157566/150544077-d76db9fa-7819-4fbd-8d19-7f2725dd0bb6.png">

**Comment by tischi** [comment]
...ideal would be if it would only show the `MoBIE - BigDataViewer` sources. I think this should be possible by means of some `SourceAndConverterProvider` for the `BdvPlaygroundActionCommand`, is it? I don't remember the details...
